playbooks/Makefile

```diff
@@ -10,6 +12,38 @@ monitoring:
 	@@if test -z "$$ENVIRONMENT"; then echo "ERROR: ENVIRONMENT is not defined."; exit 1; fi;
 	@ansible-playbook -i inventory monitoring.yml --extra-vars "@../../edx-configs/$$CUSTOMER/$$ENVIRONMENT/server-vars.yml"
 
+## Deploy testdrive
+deploy-testdrive:
+	$(eval EDXCONFIGS_REPO_LOCAL_BRANCH := $(shell cd ../../edx-configs && git symbolic-ref --short -q HEAD))
+	@appsembler/scripts/check_git_is_up2date.sh
+	@ansible-playbook --inventory-file=../../edx-configs/appsembler/testdrive/hosts/inventory testdrive.yml --ask-vault-pass
```

Comment:
As I mentioned [here](https://github.com/noderabbit-team/edx-configs/pull/7#pullrequestreview-538250), it would be good to have separate production and staging settings. Not sure what would be the best way of adding that to the Makefile, either a separate command or some kind of a parameter?